staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -25,8 +25,7 @@ import (
 
 	"k8s.io/utils/clock"
 
-	"k8s.io/apimachinery/pkg/util/runtime"
-	"k8s.io/apiserver/pkg/util/flowcontrol/counter"
+	_ "k8s.io/apiserver/pkg/util/flowcontrol/counter" // hack to keep test code in vendor
```

Comment:
What testcode do you need in vendor?

Reply:
I suspect this one is no longer needed.  I have deleted it.